### DISCUSSION TITLE
Integrations to Notification services

### DISCUSSION THREAD

**Discussion by mrlfarano** [jdholtz/auto-southwest-check-in#322] [Ideas]
Hi there,
I love your project, very well done. I wanted to add, an important component is notifications.
Do you have any plans to include notification services like Pushover, Discord, Prowl, etc?

**Comment by jdholtz** [comment]
Hi, thank you. Yes, the script can be set up to send notifications through almost any notification service using [Apprise](https://github.com/caronc/apprise). Here’s a link to how you can do that in your configuration file: https://github.com/jdholtz/auto-southwest-check-in/blob/master/CONFIGURATION.md#notifications

If you want to request support for more services, I would file an issue in Apprise’s repository.
